tools/dirchange/CMakeLists.txt

```diff
@@ -7,5 +7,5 @@
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}")
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 
-add_executable(dirchange EXCLUDE_FROM_ALL dirchange)
+add_executable(dirchange.cpp EXCLUDE_FROM_ALL dirchange)
```

Comment:
This is renaming the executable to `dirchange.cpp`, not changing file reference.

Reply:
Woops